pkg/container/container.go

```diff
@@ -157,6 +162,19 @@ func (c *containerModule) Run(ns string, data pkg.Container) (id pkg.ContainerID
 		return id, ErrEmptyRootFS
 	}
 
+	startupFile, err := os.Open(filepath.Join(data.RootFS, ".startup.toml"))
+	if err == nil {
+		defer startupFile.Close()
+
+		log.Info().Msg("startup file found, extracting environment variable")
+		envs, err := readEnvs(startupFile)
+		if err != nil {
+			return id, errors.Wrap(err, "failed to extract environment variable from startup file")
+		}
+
+		data.Env = mergeEnvs(envs, data.Env)
+	}
+
```

Comment:
I see that u still do not use the `exec` from the `startup.toml` as an entrypoint for the container.
The procedure should be the following
- check if .startup.toml exists (opens with no error)
- load the `entry` point.
- if an `entrypoint` is already provided by the reservation, use it.
- if no `entrypoint` is provided, use the one from .startup.tom
- update the env